src/sentry/profiles/task.py

```diff
@@ -204,8 +208,15 @@ def process_profile_task(
         except Exception as e:
             sentry_sdk.capture_exception(e)
 
-    if not _push_profile_to_vroom(profile, project):
-        return
+    if (
+        features.has("projects:continuous-profiling-vroomrs-processing", project)
+        and "profiler_id" in profile
```

Comment:
Just checking but does vroomrs handle android chunks already?

Reply:
Correct, vroomrs fully supports profile chunks (Android and Sample format).